### PR TITLE
Revert "Ericmdantas master"

### PR DESCRIPTION
**Pull request by georgeedwards**
Reverts georgeedwards/Gen-App#5

### CODE REVIEW

**Comment by georgeedwards** [comment]
Sorry, was just messing around, I'll wait till we have this building before merging

**Comment by ericmdantas** [comment]
Why revert the change to the package.json, though? https://github.com/georgeedwards/Gen-App/blob/master/package.json#L9. The `babel-core` one, I mean.

**Comment by georgeedwards** [comment]
@ericmdantas Sorry, I thought that was one of my attempts to fix that problem - I have re-commited that now

**Comment by georgeedwards** [comment]
@ericmdantas is there anything I can do to help with this?

**Comment by ericmdantas** [comment]
@georgeedwards, sure. You can fix paths and suchs that could be broken. We'll also need a method to clean the db after each test, that's what the `Todo.remove()` was supposed to do.

**Comment by georgeedwards** [comment]
@ericmdantas I have tried to change the two unit test references to the DAO functions in the server project, but it is still failing, am I trying to do the right thing, or have I misunderstood what's going on? I am not familiar with Chai.

File: tests/server/_helpers/db.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var Todo = require('../../../server/api/Private/todo/dao/todo-dao').default;
+var Todo = require('../../../server/api/Private/todo/dao/todo-dao.js');
```
Comment (ericmdantas) [inline tests/server/_helpers/db.js]: this should still get the `.default`

File: tests/server/todo/daos/todo-dao_test.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 var mongoose = require('mongoose');
-var TodoDAO = require('../../../../server/api/Private/todo/dao/todo-dao').default;
+var TodoDAO = require('../../../../server/api/Private/todo/dao/todo-dao.js');
```
Comment (ericmdantas) [inline tests/server/todo/daos/todo-dao_test.js]: same here

File: tests/server/todo/daos/todo-dao_test.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ describe('todo.dao', () => {
     });
 
     afterEach(() => {
-        TodoDAO.remove();
+        TodoDAO.deleteTodo();
```
Comment (ericmdantas) [inline tests/server/todo/daos/todo-dao_test.js]: this should be `remove`, see http://mongoosejs.com/docs/api.html#model_Model-remove

**Comment by ericmdantas** [comment]
Added some comments.

I'll get home soon and help you out with this.

**Comment by ericmdantas** [comment]
I got the server tests to run locally with my repo: https://github.com/ericmdantas/Gen-App. Do you want me to open a new PR? Not sure how I'd commit to this one.

**Comment by georgeedwards** [comment]
Yes, please open a new PR.

On 3 Apr 2016, at 03:19, Eric M. Dantas <notifications@github.com<mailto:notifications@github.com>> wrote:


I got the server tests to run locally with my repo: https://github.com/ericmdantas/Gen-App. Do you want me to open a new PR? Not sure how I'd commit to this one.

-

**Pull request closed without merging**